index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title>Document</title>
```

Comment:
- [ ] Based on the project requirement, you are expected to make _**at least two comments**_ about a bug or an issue with the code. You were supposed to introduce an obvious issue and then give a review comment on that issue. Kindly add these comments. 
   - For example, Your README file has a lot of issues that you could use to speak on. 
   - Note - You can more than two comments but not less than two comments.
- [ ] You are expected to use [GitHub feature inline comment](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) at least once to give a code review. Please read [this article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) to learn how to do a code review.